src/extension.ts

```diff
@@ -113,6 +114,12 @@ export function activate(context: ExtensionContext) {
         return text;
       };
 
+      /*if (extensions.onDidChange) {// Theia doesn't support this API yet
```

Comment:
if Theia doesn't support it, then extensions.onDidChange should be false, so you wouldn't go into the block. So why is this block commented?

Reply:
I commented that because I had some compilation problem (because vscode @types was not used). Now we use @types it works better, I uncomment this code.